editor/editor_toaster.cpp

```diff
@@ -28,13 +28,13 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 
+#include "editor/editor_toaster.h"
```

Comment:
```suggestion
#include "editor_toaster.h"
```
For the header corresponding to this `.cpp` we don't use an absolute path normally.